instrumentation/jetty-httpclient/jetty-httpclient-9.2/library/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/httpclient/v9_2/JettyClientHttpAttributesExtractor.java

```diff
@@ -0,0 +1,150 @@
+      String queryString = request.getQuery();
+      return queryString != null ? request.getPath() + "?" + queryString : request.getPath();
+    }
+    return null;
+  }
+
+  @Override
+  @Nullable
+  protected String host(Request request) {
+    return request != null ? request.getHost() : null;
+  }
+
+  @Override
+  @Nullable
+  protected String route(Request request) {
+    return null;
+  }
+
+  @Override
+  @Nullable
+  protected String scheme(Request request) {
+    return request != null ? request.getScheme() : null;
+  }
+
+  @Override
+  @Nullable
+  protected String userAgent(Request request) {
+    if (request != null) {
+      HttpField agentField = request.getHeaders().getField(HttpHeader.USER_AGENT);
+      return agentField != null ? agentField.getValue() : null;
+    }
+    return null;
+  }
+
+  @Override
+  @Nullable
+  protected Long requestContentLength(Request request, @Nullable Response response) {
+    Long reqContentLength = null;
+    if (request != null) {
+      HttpField requestContentLengthField =
+          request.getHeaders().getField(HttpHeader.CONTENT_LENGTH);
+      reqContentLength = getLongFromJettyHttpField(requestContentLengthField);
+    }
+    return reqContentLength;
+  }
+
+  @Override
+  @Nullable
+  protected Long requestContentLengthUncompressed(Request request, @Nullable Response response) {
+    return null;
+  }
+
+  @Override
+  @Nullable
+  protected String flavor(Request request, @Nullable Response response) {
+
+    if (response != null) {
+      HttpVersion httpVersion = response.getVersion();
+      for (String version : httpFlavors) {
+        if (httpVersion.toString().endsWith(version)) {
+          return version;
+        }
+      }
+    }
+    return "1.1";
+  }
+
+  @Override
+  @Nullable
+  protected String serverName(Request request, @Nullable Response response) {
+    return null;
+  }
+
+  @Override
+  @Nullable
+  protected String clientIp(Request request, @Nullable Response response) {
+    return null;
+  }
+
+  @Override
+  @Nullable
+  protected Integer statusCode(Request request, Response response) {
+    return response != null ? response.getStatus() : null;
+  }
+
+  @Override
+  @Nullable
+  protected Long responseContentLength(Request request, Response response) {
+    Long respContentLength = null;
+    if (response != null) {
+      HttpField requestContentLengthField =
+          response.getHeaders().getField(HttpHeader.CONTENT_LENGTH);
+      respContentLength = getLongFromJettyHttpField(requestContentLengthField);
+    }
+    return respContentLength;
+  }
+
+  @Override
+  @Nullable
+  protected Long responseContentLengthUncompressed(Request request, Response response) {
+    return null;
+  }
+
+  private static Long getLongFromJettyHttpField(HttpField httpField) {
+    Long longFromField = null;
+    try {
+      longFromField = httpField != null ? Long.getLong(httpField.getValue()) : null;
+    } catch (NumberFormatException t) {
+      LOG.error(
```

Comment:
Consider using debug:
```suggestion
      LOG.debug(
```
The http client library will probably detect invalid header values and log something by itself, the instrumentation does not need to add more noise to the user's log output.